tests/playwright/utils/consts.ts

```diff
@@ -7,3 +7,8 @@ export const stateAbbreviation = process.env.TEST_STATE || "MN";
 export const stateName = process.env.TEST_STATE_NAME || "Minnesota";
 
 export const currentYear: number = new Date().getFullYear();
+
+export const newBannerTitle: string = "Newly Created Banner";
```

Comment:
Are we going to need these consts in other tests? If it's just for `createAdminBanner`, then I'd be fine with just hardcoding the values there as needed.